testsuites/store/src/main/java/org/eclipse/rdf4j/sail/RDFStoreTest.java

```diff
@@ -718,6 +726,7 @@ public void testQueryBindings()
 	}
 
 	@Test
+	@Ignore("test is fundamentelly flawed as it uses multithreaded access on the same connection")
```

Comment:
Yeah, nah. The test seems to be specifically about the behavior of things like the `close()` method when using the same connection in multiple threads. Let's leave it as ignored, or even just remove it.